### PR TITLE
Feyre frontend

### PR DESCRIPTION
**Pull request by GrantBirki**
# Feyre Frontend 🚪

This PR merges the `Feyre Frontend` into the `working` branch so that the frontend components may be used with the `backend`.

## What is this?

This PR creates the `Feyre Frontend` using the Discord.js library with Slash commands.

## Why is this?

To separate the front and backend to be their own microservice. This allows for them to be scaled independently and for us to work in Python on the backend rather than in nodeJS.

## Important changes 💡

- Rather than running `make run` in subfolders, you just run `make run` once from the root of the repo and it builds all services (including the frontend!)

## Notices

Please check the following areas to make sure things are still working as expected:

- Docker volumes - Make sure they still work for ya
- Docker builds - Make sure you can run `make run` and everything spins up correctly

## A note on scaling (frontend) ⚙

I spent a good chunk of time today debugging JS to try and find where an error was being raised from when attempting to run two frontend containers at a time.

Rather than just run one frontend container, it seems it would be a good idea to be able to run multiple in parallel so we can scale horizontally. When attempting to do so, one container serves the request and the other container crashes. This is probably due to the first container processing the request and then the other one panicking because it doesn't know what to do with a request that has already been completed. I tried putting `try/except` statements all over with no success.

In short, we will need to do JS wizardry 🧙 in order to catch the exception and return so we can run two containers at once without lots of crashes.

### CODE REVIEW

**Comment by iancaragol** [comment]
Looks great, containers started up with no issues :)
